arangod/GeneralServer/CommTask.cpp

```diff
@@ -516,7 +515,6 @@ void CommTask::executeRequest(std::unique_ptr<GeneralRequest> request,
     return;
   }
 
-  TRI_ASSERT(SchedulerFeature::SCHEDULER != nullptr);
```

Comment:
nit: I think the assertion is still valid. I would vote to keep it.

Reply:
Thanks! The assertions got lost because I previously had another change which affected the scheduler, and when I reverted those and removed the changes I apparently also removed those assertions. Will check all places and restore them.